mlflow/sagemaker/cli.py

```diff
@@ -17,20 +17,40 @@ def commands():
 
 
 @commands.command("deploy")
-@click.option("--app-name", "-a", help="Application name.", required=True)
+@click.option("--app-name", "-a", help="Application name", required=True)
 @cli_args.MODEL_PATH
 @click.option("--execution-role-arn", "-e", default=None, help="SageMaker execution role")
 @click.option("--bucket", "-b", default=None, help="S3 bucket to store model artifacts")
 @cli_args.RUN_ID
 @click.option("--image-url", "-i", default=None, help="ECR URL for the Docker image")
-@click.option("--region-name", "-r", default="us-west-2", help="region name")
-def deploy(app_name, model_path, execution_role_arn, bucket, run_id, image_url, region_name):
+@click.option("--region-name", "-r", default="us-west-2", 
+              help="Name of the AWS region in which to deploy the application")
+@click.option("--mode", "-md", default=mlflow.sagemaker.DEPLOYMENT_MODE_CREATE, 
+              help="The mode in which to deploy the application")
```

Comment:
would be helpful to list the options

Reply:
Changed the help statement to the following: 

```
help="The mode in which to deploy the application. Must be one of the modes listed in mlflow.sagemaker.DEPLOYMENT_MODES"
```